docs/configuration_files.md

```diff
@@ -79,8 +79,10 @@ Document the configuration options of PokemonGo-Bot.
 | `max_steps`        | 5       | The steps around your initial location (DEFAULT 5 mean 25 cells around your location) that will be explored
 | `forts.avoid_circles`             | False     | Set whether the bot should avoid circles |
 | `forts.max_circle_size`             | 10     | How many forts to keep in ignore list |
-| `walk_max`             | 4.16    | Set the maximum walking speed in m/s (1 is about 3.6km/hr) 4.16m/s = 15km/h
-| `walk_min`             | 2.16    | Set the minimum walking speed in m/s (1 is about 3.6km/hr) 2.16m/s = 7.8km/h
+| `walk_max`             | 4.16    | Set the maximum walking speed in m/s (1 is about 3.6km/h) 4.16m/s = 15km/h
+| `walk_min`             | 2.16    | Set the minimum walking speed in m/s (1 is about 3.6km/h) 2.16m/s = 7.8km/h
+| `fly_max`              |  282    | Set the maximum flying speed in m/s (1 is about 3.6km/h) 282m/s = 1015km/h
```

Comment:
You're right, I got those numbers from somewhere, but a fresh search shows your numbers are more realistic